core/src/main/java/com/graphhopper/storage/change/ChangeGraphHelper.java

```diff
@@ -93,7 +93,7 @@ private long applyChange(JsonFeature jsonFeature, FlagEncoder encoder) {
         BooleanEncodedValue accessEnc = encoder.getAccessEnc();
         DecimalEncodedValue avSpeedEnc = encoder.getAverageSpeedEnc();
         long updates = 0;
-        EdgeFilter filter = DefaultEdgeFilter.allEdges(encoder);
+        EdgeFilter filter = EdgeFilter.ALL_EDGES;
```

Comment:
@boldtrn do you think this is a reasonable fix?

Reply:
I actually had the same idea to approach this issue. 

The biggest issue I see with this is that when we try to block an edge, we might snap the wrong edge. For example one wants to avoid a road and puts an avoidance on that road, we might incorrectly snap for example the cycle way next to the road, that has no relevance for the car. Other than that I think this can work :+1:.